pkg/github/pull_request_reviews.go

```diff
@@ -0,0 +1,244 @@
+		data.NewField("pull_request_author_company", nil, []string{}),
+		data.NewField("repository", nil, []string{}),
+		data.NewField("review_author_name", nil, []string{}),
+		data.NewField("review_author_login", nil, []string{}),
+		data.NewField("review_author_email", nil, []string{}),
+		data.NewField("review_author_company", nil, []string{}),
+		data.NewField("review_url", nil, []string{}),
+		data.NewField("review_state", nil, []string{}),
+		data.NewField("review_comment_count", nil, []int64{}),
+		data.NewField("review_updated_at", nil, []time.Time{}),
+		data.NewField("review_created_at", nil, []time.Time{}),
+	)
+
+	for _, pr := range prs {
+		for _, review := range pr.Reviews {
+			frame.AppendRow(
+				pr.Number,
+				pr.Title,
+				string(pr.State),
+				pr.URL,
+				pr.Author.User.Name,
+				pr.Author.User.Login,
+				pr.Author.User.Email,
+				pr.Author.User.Company,
+				pr.Repository.NameWithOwner,
+				review.Author.User.Name,
+				review.Author.User.Login,
+				review.Author.User.Email,
+				review.Author.User.Company,
+				review.URL,
+				string(review.State),
+				review.CommentsCount,
+				review.UpdatedAt.Time,
+				review.CreatedAt.Time,
+			)
+		}
+	}
+
+	return data.Frames{frame}
+}
+
+// GetAllPullRequestReviews uses the graphql search endpoint API to search all pull requests in the repository
+// and all reviews for those pull requests.
+func GetAllPullRequestReviews(ctx context.Context, client models.Client, opts models.ListPullRequestsOptions) (PullRequestReviews, error) {
+	var (
+		variables = map[string]interface{}{
+			"prCursor":     (*githubv4.String)(nil),
+			"reviewCursor": (*githubv4.String)(nil),
+			"query":        githubv4.String(buildQuery(opts)),
+		}
+
+		pullRequestReviews = PullRequestReviews{}
+	)
+
+	for {
+		q := &QueryListPullRequestReviews{}
+		if err := client.Query(ctx, q, variables); err != nil {
+			return nil, errors.WithStack(err)
+		}
+
+		prs := make([]PullRequestWithReviews, len(q.Search.Nodes))
+
+		for i, prNode := range q.Search.Nodes {
+			pr := prNode.PullRequest
+
+			prs[i] = PullRequestWithReviews{
+				Number:     pr.Number,
+				Title:      pr.Title,
+				State:      pr.State,
+				URL:        pr.URL,
+				Author:     pr.Author,
+				Repository: pr.Repository,
+			}
+
+			for {
+				for _, reviewNode := range pr.Reviews.Nodes {
+					review := reviewNode.Review
+
+					prs[i].Reviews = append(prs[i].Reviews, Review{
+						CreatedAt:     review.CreatedAt,
+						UpdatedAt:     review.UpdatedAt,
+						URL:           review.URL,
+						Author:        review.Author,
+						State:         review.State,
+						CommentsCount: review.Comments.TotalCount,
+					})
+				}
+
+				if !pr.Reviews.PageInfo.HasNextPage {
+					variables["reviewCursor"] = (*githubv4.String)(nil)
+					break
+				}
+
+				variables["reviewCursor"] = pr.Reviews.PageInfo.EndCursor
+				if err := client.Query(ctx, q, variables); err != nil {
+					return nil, errors.WithStack(err)
+				}
```

Comment:
Majority of cases this'll never happen, but covering the case of more than 100 reviews